src/sentry/experiments/__init__.py

```diff
@@ -5,3 +5,4 @@
 manager = ExperimentManager()
 
 all = manager.all
+get = manager.get
```

Comment:
I don't quite understand why we have the pattern of putting the manager methods as module variables here. I think you'd be better off accessing them as `manager.<method>`